app/models/casa_case.rb

```diff
@@ -133,10 +133,6 @@ def latest_past_court_date
     past_court_dates.order("date").last
   end
 
-  def has_contact_type?(contact_type)
-    casa_case_contact_types.map(&:contact_type_id).include?(contact_type.id)
-  end
```

Comment:
I moved this to `casa_case_decorator.rb` so I could call `model.decorate.has_contact_type?` for both `case_contacts` and `casa_cases`